src/client/include/client/system/SystemRenderGUI.hpp

```diff
@@ -4,15 +4,68 @@
 #include <irrlicht.h>
 #include <tempo/time.hpp>
 #include <tempo/component/ComponentHealth.hpp>
+#include <ctime>
 
 namespace client
 {
 struct SystemRenderGUI {
-	void update(irr::video::IVideoDriver * driver,
-	            irr::gui::IGUIEnvironment *gui_env,
-	            tempo::Clock &             clock,
-	            int                        combo,
-						  tempo::ComponentHealth     comp_health);
+
+	private:
+		// Misc Colours
+		irr::video::SColor colour_white = irr::video::SColor(255, 255, 255, 255);
+		irr::video::SColor colour_red   = irr::video::SColor(255, 255, 0, 0);
+		irr::video::SColor colour_green = irr::video::SColor(255, 0, 255, 0);
+		irr::video::SColor colour_blue  = irr::video::SColor(255, 135, 206, 250);
```

Comment:
We should look into these at some point - the pure colours are not that good looking

Reply:
That is assuming they're being used as is